src/h5p-helpers/render.js

```diff
@@ -17,32 +17,30 @@ exports.render = editor => {
             <meta charset="utf-8">
             <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
             <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.css" crossorigin="anonymous">
+            <script src="https://cdn.jsdelivr.net/npm/clipboard@2.0.8/dist/clipboard.min.js"></script>
             <title>Course H5P Library</title>
         </head>
         <body>
             <div class="container" style="margin-top:60px;">
                 <h1>Course H5P Library</h1>
-                ${
-                  req.session.context_title
-                    ? `<div class="alert alert-warning">
+                ${req.session.context_title
+        ? `<div class="alert alert-warning">
                       Content shown only for the course: 
                       ${req.session.context_title}
                     </div>`
-                    : ""
-                }
-                <a class="btn btn-primary my-2" href="${
-                  editor.config.baseUrl
-                }/new" target="_blank"><span class="fa fa-plus-circle m-2"></span>Create new content</a>
-                ${
-                  filteredContentObjects.length == 0
-                    ? "<h4 style='margin:30px 0'>There's no H5P content for this course yet</h4>"
-                    : "<h2 style='margin: 30px 0'>Existing content</h2>"
-                }
+        : ""
+      }
+                <a class="btn btn-primary my-2" href="${editor.config.baseUrl
+      }/new" target="_blank"><span class="fa fa-plus-circle m-2"></span>Create new content</a>
+                ${filteredContentObjects.length == 0
+        ? "<h4 style='margin:30px 0'>There's no H5P content for this course yet</h4>"
+        : "<h2 style='margin: 30px 0'>Existing content</h2>"
+      }
                 <div class="list-group">
                 ${filteredContentObjects
-                  .map(
-                    content =>
-                      `<div class="list-group-item">
+        .map(
```

Comment:
Where do these whitespace changes come from? Are they done by `prettier`?

Reply:
Yes. Unfortunately the prettier messed it up.

Reply:
That's good. Let's follow the code style that prettier provides.